### PR TITLE
[ALLUXIO-1589] Added test coverage for KeyValueStore factories

### PR DESCRIPTION
**Pull request by HorizonNet**
https://alluxio.atlassian.net/browse/ALLUXIO-1589
- Added tests for the factory of KeyValuePartitionWriter
- Added tests for the factory of KeyValuePartitionReader
- Added tests for the factory of KeyValueSystem

### CODE REVIEW

**Comment by HorizonNet** [comment]
alluxio-bot, check this please

File: tests/src/test/java/alluxio/client/keyvalue/KeyValuePartitionIntegrationTest.java [hunk 1]
```diff
@@ -172,4 +179,66 @@ public void noOrderIterator() throws Exception {
     Collections.sort(iteratedPairs);
     Assert.assertEquals(pairs, iteratedPairs);
   }
+
+  /**
+   * Tests that an exception is thrown when trying to create an instance via
+   * {@link alluxio.client.keyvalue.KeyValuePartitionWriter.Factory#create(AlluxioURI)} which is
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/client/keyvalue/KeyValuePartitionIntegrationTest.java]: Do we need the fully qualified path here?

File: tests/src/test/java/alluxio/client/keyvalue/KeyValuePartitionIntegrationTest.java [hunk 2]
```diff
@@ -172,4 +179,66 @@ public void noOrderIterator() throws Exception {
...
+  @Test
+  public void createWriter() throws IOException, AlluxioException {
+    mKeyValuePartitionWriter = KeyValuePartitionWriter.Factory.create(mPartitionUri);
+    Assert.assertNotNull(mKeyValuePartitionWriter);
+    mKeyValuePartitionWriter.close();
+  }
+
+  /**
+   * Tests that an exception is thrown when trying to create an instance via
+   * {@link alluxio.client.keyvalue.KeyValuePartitionReader.Factory#create(AlluxioURI)} which is
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/client/keyvalue/KeyValuePartitionIntegrationTest.java]: Do we need the fully qualified path here?

**Comment by HorizonNet** [comment]
@aaudiber Thanks for the comments. I updated the PR accordingly.

**Pull request merged**